sources/Inputs/include/AssemblingDataBase.h

```diff
@@ -418,7 +426,8 @@ class AssemblingDataBase {
   std::unordered_map<std::string, MultipleAssociation> multipleAssociations_;       ///< list of multiple associations
   std::unordered_map<std::string, std::string> generatorIdToSingleAssociationsId_;  ///< generator id associated to the single associations that contains it
   std::map<std::string, DynamicAutomaton> dynamicAutomatons_;                       ///< list of dynamic automatons
-  std::unordered_map<std::string, Property> properties_;                            ///< list of properties
+  bool containsSVC_;                                      ///< true if the assembling data base contains one or more SVC, false otherwise
```

Comment:
comment alignment

Reply:
The comments were aligned by clang-format